src/components/Footer/index.tsx

```diff
@@ -116,6 +120,16 @@ export function Footer({
                 {i18n.brand}
               </a>
             ) : null}
+            {flags.showCareersLink ? (
+              <a
+                className={FooterSectionStyle.footer__link}
+                href="https://jobs.ashbyhq.com/windranger?departmentId=d4a6dd89-7856-4045-921b-e982d346249c"
```

Comment:
could we make this href a prop so that we can update it in the future without sending a UI PR?

That way the UI component is slightly more stateless/un-opinionated. 

This has been useful in places like the nav bar in the past.